source/dofs/dof_tools_sparsity.cc

```diff
@@ -1390,16 +1407,22 @@ namespace DoFTools
              "associated DoF handler objects, asking for any subdomain other "
              "than the locally owned one does not make sense."));
 
+    Assert(
+      face_has_flux_coupling,
+      ExcMessage(
+        "The function which specifies if a flux coupling occurs over a given "
+        "face is empty."));
+
     internal::make_flux_sparsity_pattern(dof,
```

Comment:
To work around the MSVC issue, try to specify the template arguments explicitly here:
```suggestion
    internal::make_flux_sparsity_pattern<dim,spacedim>(dof,
```
Let's see whether that helps!

Reply:
I tried this but it didn't help.